pyproject.toml

```diff
@@ -124,6 +125,7 @@ good-names = [
 minversion = "7.0"
 addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --doctest-ignore-import-errors"
 filterwarnings = [
+    "error",
```

Comment:
If I understand it correctly, this one turns warnings into errors. 

If we do this, all warnings in the tests should either be explicitly ignored or catched with something like this:

```python
def test_warning():
    with pytest.warns(UserWarning):
        warnings.warn("my warning", UserWarning)
```